From: <saigkill@opensuse.org>
Date: 2011-04-28 17:29:54 +0000
X-Upstream: no

Create password file rw for group, this enables passwordless connection
of manager from users of the boinc group. This won't be probably
upstreamed as it might be unsafe for common usage without setting
proper group ownership of the password file.

---
 client/gui_rpc_server.cpp |    7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

Index: boinc-client_release-7.8-7.8.3/client/gui_rpc_server.cpp
===================================================================
--- boinc-client_release-7.8-7.8.3.orig/client/gui_rpc_server.cpp
+++ boinc-client_release-7.8-7.8.3/client/gui_rpc_server.cpp
@@ -176,12 +176,7 @@ void GUI_RPC_CONN_SET::get_password() {
     // they can cause code to execute as this user.
     // So better protect it.
     //
-    if (g_use_sandbox) {
-        // Allow group access so authorized administrator can modify it
-        chmod(GUI_RPC_PASSWD_FILE, S_IRUSR|S_IWUSR | S_IRGRP | S_IWGRP);
-    } else {
-        chmod(GUI_RPC_PASSWD_FILE, S_IRUSR|S_IWUSR);
-    }
+    chmod(GUI_RPC_PASSWD_FILE, S_IRUSR|S_IWUSR | S_IRGRP | S_IWGRP);
 #endif
 }
 
